app/components/nav-rail.tsx

```
@@ -6,76 +6,100 @@ import NavRailPinnedItem from './nav-rail-pinned-item';
+                                    onClick={() => { setSelected(index) }}
+                                    extended={extended}
+                                />
+                        })
+                    }
+                </div>
+                <hr />
+                <div className="nav-rail-pinned flex-col">
+                    {
+                        props.pinned !== undefined
+                        ? Object.keys(props.pinned).map((key: string, index: number) => {
+                            if (props.pinned![key]['img']!['src'] === undefined) {
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)